docs/intermediate/least_to_most.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 5
+locale: en-us
+style: chicago
+---
+
+# 🟢 Least to Most Prompting
```

Comment:
I think this should be yellow, since this method requires a small amount of programming between steps. What do you think?

Reply:
I thought about that. It only requires programming in the SCAN case. Happy to turn it yellow.

Reply:
I left it green for now (?).